### ISSUE TITLE
Generate event-time dummies without estimations

### ISSUE THREAD

**Issue by Constantino-Carreto-Romero** [JMSLab/xtevent#23]
Per call: An option that generates event-time dummies without estimating the model. This might be useful if the users want to customize their regressions and plots.

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero  will work on this, seems quite easy. We'll want something akin to a `noestimate` option.

**Comment by Constantino-Carreto-Romero** [comment]
@jorpppp There already exists the option `savek(stub)` that saves the evet-time dummies after estimating the model. This option would be related to the option `noestimate`, both options save the event-time dummies, the only difference is if the model is estimated. I was thinking of rather implementing `noestimate` as a suboption embeded in `savek`, and the default behavior for `savek` would be to save the event-time dummies and estimate the model, while specifying `savek(stub, noestimate)` it would save the event-time dummies but without estimating the model. What do you think of this?

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero, yes, sounds good.

**Comment by Constantino-Carreto-Romero** [comment]
@jorpppp I have implemented the suboption `noestimate`. I tested its functionality with some code of my own and I also ran the test file to check if something was broken. 
Some examples could be testes running the following:
```stata
net install xtevent, from("https://raw.githubusercontent.com/JMSLab/xtevent/issue23_gen_variables_without_estimation") replace
use "https://github.com/JMSLab/xtevent/blob/main/test/example31.dta?raw=true", clear
xtset i t
*default
xtevent y, policyvar(z) window(3) 
*estimate
xtevent y, policyvar(z) window(3) savek(aa)
*don't estimate
cap drop aa*
xtevent y, policyvar(z) window(3) savek(aa, noe)
*interaction with other functions
cap drop aa*
xtevent y, policyvar(z) window(3) savek(aa) trend(-3, method(ols) saveov)
```

If this branch is merged before the release, the release version should be 2.2.0,

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero thanks, we won't release this yet, let me test it first.

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero Thanks for this. I left some comments in https://github.com/JMSLab/xtevent/commit/513bee1e1ab4ad4d53d4b07fd926c76966cda789 for you to check, please.

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero I checked the answers to my comments, thanks. Can you add examples to the test file and the help files, please, and re-run the examples in the helpfiles and the test file to make sure nothing broke, please?

After that, we can start a PR.

**Comment by Constantino-Carreto-Romero** [comment]
@jorpppp I left an extra comment in https://github.com/JMSLab/xtevent/commit/513bee1e1ab4ad4d53d4b07fd926c76966cda789#r81256048 if you are fine with no changing what I mentioned on that comment I will start the PR after adding the examples. Thanks

**Comment by Constantino-Carreto-Romero** [comment]
@jorpppp I modified the example of `savek(stub)` to an example of `savek(stub, noestimate)` in the help file in commit https://github.com/JMSLab/xtevent/commit/d2d4c81e4866dfc63411b6aba3329cda93d93e8d. 
There were a couple of code lines to which I had not implemented the click functionality in issue #84  because they contained a colon and this was problematic, but I already found a way to make them click-able despite of this and I also implemented this in https://github.com/JMSLab/xtevent/commit/d2d4c81e4866dfc63411b6aba3329cda93d93e8d
I added an example to the test file in https://github.com/JMSLab/xtevent/commit/0f96acb4f98f25fed15e822231485d8a2468ebe1.
I checked that the examples in the help file and the test file ran properly, as well.
